src/crm.ts

```diff
@@ -0,0 +1,113 @@
+import { ActionMessageV2, EventMessageV2 } from "@proca/queue";
+export type handleResult = {
+  pause?: 0; // should it wait delay seconds before reading the next message?
+  processed: boolean; // if the message has been processed or skipped
+};
+
+export { ActionMessageV2 as ActionMessage };
+export { EventMessageV2 as EventMessage };
+
+type Obj = { [key: string]: any };
```

Comment:
you could just use  `Record<string,any>` - Obj is shorter but also confusing

Reply:
changed